BaseLib/filesystem.h.in

```diff
@@ -0,0 +1,3 @@
+// Provides namespace fs for std::filesystem
+#include <@CXX_FILESYSTEM_HEADER@>
+namespace fs = @CXX_FILESYSTEM_NAMESPACE@;
```

Comment:
Just a question. This is needed because sometimes the namespace is std::experimental::filesystem and sometimes std::filesystem, isn't it?

Reply:
Correct.